core/src/main/resources/lib/form/checkbox.jelly

```diff
@@ -70,10 +71,8 @@ THE SOFTWARE.
          checkUrl="${attrs.checkUrl}" checkDependsOn="${attrs.checkDependsOn}" json="${attrs.json}"
          disabled="${readOnlyMode ? 'true' : null}"
          checked="${value ? 'true' : null}"/>
-  <j:if test="${attrs.title!=null}">
-    <label class="attach-previous"
-           title="${attrs.tooltip}">${attrs.title}</label>
-  </j:if>
+  <label class="attach-previous setting-checkbox ${attrs.title == null ? 'js-checkbox-label-empty' : ''}"
```

Comment:
Why did we lose the `null` check here?

Reply:
When `<f:checkbox>` has no label, there's a JS function that will crawl the DOM tree up trying to find one and it will insert it in that empty label.

Reply:
https://github.com/jenkinsci/jenkins/commit/46b69379f10ef3b54c2fd61fc96dd0f83902b4ad#diff-da911448bbb85d374f604da9c1e00136